neqo-crypto/build.rs

```diff
@@ -152,6 +152,9 @@ fn build_nss(dir: PathBuf) {
         build_nss.push(String::from("-j"));
         build_nss.push(d);
     }
+    if env::var("TARGET").unwrap() == "aarch64-apple-darwin" {
+        build_nss.push(String::from("--target=arm64"));
+    }
```

Comment:
For this, we should be parsing the `$TARGET` triple to get the first token.  It might be better to do:

```suggestion
    let target = env::var("TARGET").unwrap();
    if target.strip_prefix("aarch64-").is_some() {
        build_nss.push(String::from("--target=arm64"));
    }
```